gpii/node_modules/solutionsRegistry/src/solutions/com.bdigital/easit4all.json5

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "Easit4all",
```

Comment:
Did this not get axed as part of the destruction of the "web" SR? I notice that @klown did not axe the corresponding test preferences easit1.json5 and east2.json5, which we should do too

Reply:
@amb26 wrote:

> @klown did not axe the corresponding test preferences easit1.json5 and east2.json5, which we should do too

My bad.  I relied on @the-t-in-rtf's validation of the test preferences, and these two sets validate because they don't list any solutions.  They are just a collection of preferences keyed by common terms.  Only the name of the preferences, "easit" harken back to the now axed solution `com.bdigital.easit4all`.  I'll check to see if there are any others like that.

Reply:
>  I'll check to see if there are any others like that.

There was a `jme_app.json5` test preference set that referenced the 'web' solution `info.cloud4all.JME`.  Both were deleted.  There remains a `jme_common.json5` preference set where the name is suspicious ("jme"), but it doesn't contain any reference to the JME solution (similar to the `easiit1` and `easit2` preferences sets).

The following preference sets contain references to the removed 'web' solution `es.codefactory.android.app.ma`:  `chris.json5`, `li.json5`, `mobileaccessibility1.json5`, and `mobileaccessibility2.json5`.

It's puzzling that these validate in the absence of the solution.  For `chris` and `li`, the solution appears inside a `metadata` block, e.g., https://github.com/GPII/universal/blob/master/testData/preferences/chris.json5#L34.  Perhaps listings within metadata are not subject to validation?  The simple fix is to remove the 'web' solution references.

For the two `mobileaccessibility` preferences sets, there are TODOs indicating that the preferences have no schema and are disabled for now -- they are commented out.

Reply:
I have not merged with upstream master in a while, I was waiting for things to settle down with the PRs in progress.

Reply:
I'm not expecting any substantial merges at this point, unless @sgithens comes up with any last-minute fixes to support capture - I think it's all yours!

Reply:
Great, I believe I've finished incorporating the updated material.  I'll have to manually prune out the web.json5 stuff as well, but that won't take long at all.

Reply:
> It's puzzling that these validate in the absence of the solution.

If there's no entry for the solution at all, then that material isn't validated.  It's a byproduct of our "per solution" validation approach that we could choose to shore up, but haven't addressed yet.